### DISCUSSION TITLE
Add to the path using Github Actions

### DISCUSSION THREAD

**Discussion by Dana-Pr** [community/community#26390] [Actions]
<p>The current  <code>add-path</code>  command will be deprecated on November 16. I tried following their new <a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#adding-a-system-path" rel="noopener nofollow ugc">instructions</a> and adding the path using  <code>echo "/path/to/dir" &gt;&gt; $GITHUB_PATH</code> , but it’s not working. What is the correct command? I am building on Windows.</p>
<p>The error that I was seeing on github actions is  <code>Error: The add-path command is deprecated and will be disabled on November 16th. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2020-10-01-github-actions-deprecating-set-env-and-add-path-commands/</code></p>

**Comment by Dana-Pr** [comment, marked as answer]
<p>Found the answer, this is what worked in Windows for me</p>
<p><code>echo "C:/Program Files (x86)/Etc" | Out-File -Append -FilePath $env:GITHUB_PATH -Encoding utf8</code></p>

**Comment by Simran-B** [comment]
<p>It depends on the shell type. The default is <code>pwsh</code> on Windows runners, so you need to do it the PowerShell way. In <code>cmd</code> it’s different:</p>
<pre><code>echo C:\Program Files (x86)\Etc &gt;&gt; %GITHUB_PATH%
</code></pre>
<p>(I checked that the GitHub runners default to chcp 65001, which is UTF-8 - so there should be no problems even with special characters in the path)</p>
